recipes/r-seurat-scripts/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = '0.0.1' %}
+
+package:
+  name: r-seurat-scripts
+  version: {{ version }}
+
+source:
+  url: https://github.com/ebi-gene-expression-group/r-seurat-scripts/archive/v{{ version }}.tar.gz
+  sha256: 580770b74194b41656e39af542af01250326f41a311038249b84dede8a57f76c
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+    run:
+        - r-base
+        - r-seurat>=2.3.1
+        - r-optparse
+        - r-workflowscriptscommon
+
+test:
+    commands:
+        - which seurat-read-10x.R
+        - which seurat-create-seurat-object.R
+        - which seurat-normalise-data.R
+        - which seurat-filter-cells.R
+        - which seurat-find-variable-genes.R
+        - which seurat-get-random-genes.R
+        - which seurat-run-pca.R
+        - which seurat-scale-data.R
+        - which seurat-dim-plot.R
+        - which seurat-find-clusters.R
+        - which seurat-run-tsne.R
+        - which seurat-find-markers.R
+        - which r-seurat-scripts-post-install-tests.sh
+        - seurat-read-10x.R --help
```

Comment:
Do you want to add this `--help` for the other tools as well? If help works, you don't need to `which` :)

Reply:
Since all the scripts load the same libraries I figured a single help would do the job of checking those, but I can change everything to helps if you think that's better- I guess it is neater.

Reply:
It's also more future-proof. Assuming you add new libraries in the future to single files.